src/passport.js

```diff
@@ -166,6 +166,8 @@ export default class Passport {
     });
   }
 
+  getEventName ()  { return this.socketEventPrefix + ev; }
```

Comment:
Indeed, I've put the argument in the last commit, the tests still failing here. I'll check again. Thanks